### PR TITLE
add Don to 'about us', add inclusivity page

### PR DESCRIPTION
**Pull request by axelsson**
* Added a separate page on our diversity and inclusivity page build from the content in this file https://docs.google.com/document/d/1-y3NjlxQ1wzxXh-tXmbXcjPSQ5yt8bUCUrCK7VpI4ek/edit# that Tove created, which is a polished version of our internal document in notion
* Added Don to 'about us' (waiting for a swedish translation of his text, gonna add that before a merge)

### CODE REVIEW

**Comment by axelsson** [comment]
[Add inclusivity page to web](https://trello.com/c/6DaXyxfF/893-add-inclusivity-page-to-web)

File: app/views/about/inclusivity/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+      This is GoClimates internal Diversity and Inclusivity Policy. It's not
+      perfect, but a start. Please drop us a line if there is anything you think
+      we could improve!
+    </p>
+
+    <h2 class="heading">Why</h2>
+    <p>
+      This is GoClimates internal Diversity and Inclusivity Policy. It's not
+      perfect, but a start. Please
+      <a class="link" href="<%=inclusivity_url%>">drop us a line</a> if there is
```
Comment (kallenilver) [inline app/views/about/inclusivity/show.html.erb:21]: Should not this be a `"mailto:hello@goclimate.com"` ?

File: public/i18n/en.json [hunk 2]
```diff
@@ -14,6 +14,7 @@
     "about_us_team": {
       "andrea": "MSc in environmental science. In love with nature and everything that it gives us. Passionate about how our society can be developed hand in hand with nature. Is constantly evolving and looking for better ways forward.",
       "cissi_html": "Cissi has a degree in Business Economics and is well versed with climate calculations and sustainability questions. Ecology nerd and horticulture genius.",
+      "don": "Environmental science graduate with a knack for customer service. Strong passion for inspiring others to take climate action. Plant-based food enthusiast who thrives in the wild.",
```
Comment (kallenilver) [inline public/i18n/en.json:17]: Vore fint att få till hans text på svenska också?

**Review by kallenilver — APPROVED** [review]
Ser bra ut!

File: public/i18n/en.json (hunk 2 above)
Comment (axelsson) [reply public/i18n/en.json:17]: Står i beskrivningen att jag väntar på den texten :)

File: app/views/about/inclusivity/show.html.erb (hunk 1 above)
Comment (axelsson) [reply app/views/about/inclusivity/show.html.erb:21]: Probably, yeah, will change.

**Pull request merged**